CHANGELOG.md

```diff
@@ -39,6 +39,7 @@
 ## 5.24.0 (2023-10-30)
 * BraintreePayPalDataCollector
   * Update PPRiskMagnes with 5.4.1 - staging removed (fixes #1107)
+  * This version of the PPRiskMagnes framework is static
```

Comment:
I am not sure this is correct, v5 uses `5.4.1-stage-removed-xcode14-1` ([Cartfile](https://github.com/braintree/braintree_ios/blob/5.x/Cartfile)). Downloading that version from [the assets](https://assets.braintreegateway.com/mobile/ios/carthage-frameworks/pp-risk-magnes/PPRiskMagnes.json) and running `file PPRiskMagnes` on both the simulator and non simulator files produces the expected dynamically linked outputs:
 ![Screenshot 2024-03-07 at 10 30 04 AM](https://github.com/braintree/braintree_ios/assets/20733831/09ce5f0a-6984-4fbc-a159-4c61be3aa736)

For static frameworks we would expect to see `current ar archive` instead of `Mach-O 64-bit dynamically linked shared library arm64` correct?

Reply:
👀 Good catch! OK, so it looks like 5.24.0 switched to static, and then 5.24.1 switches back to dynamic.

I checked this by checking out the tag version in the framework version local to the repo:
![Screenshot 2024-03-07 at 4 57 33 PM](https://github.com/braintree/braintree_ios/assets/35243507/677452c7-347b-4f88-b367-c6e4b244975c)

Reply:
Yeah exactly - iirc there was an issue with the version uploaded where it wasn't _actually_ the correct magnes version at all (I think we brought this up to magnes as a reason that we should not host their SDK for them). Here is the PR: https://github.com/braintree/braintree_ios/pull/1127. But yes, `5.24.1` is correct, and we have no idea what magnes version was in `5.24.0` because they don't include version numbers anywhere.

Reply:
I updated the PR to call out the specific difference b/w the 2 versions!